Frigate_Camera_Notifications/Beta.yaml

```diff
@@ -217,6 +217,8 @@ blueprint:
                 # format=android converts it to a 2:1 ratio for mobile display.
                 - label: Thumbnail
                   value: "{{base_url}}/api/frigate{{client_id}}/notifications/{{id}}/thumbnail.jpg"
+                - label: Review Thumbnail
+                  value: '{{base_url}}/api/frigate{{client_id}}/notifications/{{camera}}-{{review_id}}/review_thumbnail.webp'
```

Comment:
This won't work after we release a fix, as it is not great that the camera is required in that specific format. Instead it will `/notifications/{{review_id}}/{{camera}}/review_thumbnail.webp`